mail_composer_cc_bcc_account/models/mail_thread.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _message_create(self, values_list):
```

Comment:
Do we need this method? if so, it should be in mail_compose_cc_bcc instead.
(i reviewed this one, because this module was not merged yet to 17.0)

Reply:
Hi anh @nguyenminhchien, If I'm not wrong, it's needed to store cc/bcc mails, which is later used to fill in email client

https://github.com/trisdoan/social/blob/8aaec963e5ac8daac83313b07ef9266c4c2b6eb2/mail_composer_cc_bcc/models/mail_mail.py#L40-L46

Without this

1. Message does not have cc/bcc value
![mail_composer_bcc_account](https://github.com/user-attachments/assets/d3bb9b32-936e-4b67-97e7-63a4df99bb66)

2. Sent mail does not have correct values for Cc & X-Odoo-Bcc
![2024-11-11_15-27](https://github.com/user-attachments/assets/73489780-ae8c-4ba8-a5ef-8850054e733b)